README.md

```diff
@@ -1,2 +1,35 @@
-# angular-injecting-a-service-lab
-<p data-visibility='hidden'>View <a href='https://learn.co/lessons/angular-injecting-a-service-lab' title='angular-injecting-a-service-lab'>angular-injecting-a-service-lab</a> on Learn.co and start learning to code for free.</p>
+# Injecting a Service
+
+## Objectives
+
+- Create a $timeout that updates the View
+
+## Instructions
+
+Let's inject one of Angular's built in services into our controller. 
+
+For this lab, we're going to make use of `$timeout`. `$timeout` allows us to create timeouts (similar to `setTimeout`) in our applications whilst ensuring that any changes to our model done inside a timeout gets reflected in the view.
+
+`$timeout` accepts the same parameters as `setTimeout` - a function and an integer for how many milliseconds delay. 
+
+First, lets setup our directory structure:
+ 
+- js/
+ - app/
+   - controllers/
+     - ContactController.js
+   - app.js
+ - angular.js
+- index.html
+
+You can find `app.js`, `angular.js` and `index.html` in this repo.
+
+Let's create our controller inside `ContactController.js`, this should have two parameters - `$scope` and `$timeout` (it doesn't matter what order you put them in, if you don't believe me - try it!).
```

Comment:
should students be using the controllerAs syntax instead of `$scope`? I like that you're showing that the order of the parameters doesn't matter, but once we introduce a best practice we should continue to reinforce it for the students going forward